source/features/highest-rated-comment.css

```diff
@@ -1,5 +1,17 @@
+.rgh-highest-rated-comment {
+	border: 2px solid var(--color-text-success);
```

Comment:
It's working in Firefox but not in Chrome for me

<img width="1023" alt="Screen Shot 11" src="https://user-images.githubusercontent.com/1402241/125161227-549a7d00-e1ab-11eb-8c06-cda0ff550970.png">

Reply:
That's strange. I'll compile, test and get back to you

Reply:
Resolved in 2fb9338